Ktisis/Locale/i18n/fr_FR.json

```diff
@@ -55,7 +55,7 @@
 				"drawLines": "Dessiner des lignes sur le squelette",
 				"__comment": "NOTE : Ceux-la devraient refléter plus clairement la fonctionnalité réelle (dessiner des lignes/points lorsque le *gizmo est actif*)."",
 				"drawLineWhenSelecting": "Tracer des lignes lors de la sélection",
```

Comment:
```json
"drawLineWhenSelecting": "Afficher une ligne lors d'une sélection"
```
sound more in-context to me but I'm not exactly sure where the english text is displayed. Same for the one below `Afficher un point lors d'une selection`